contexts/security-v3-unstable.jsonld

```diff
@@ -1,170 +1,531 @@
 {
   "@context": [
     {
-      "@version": 1.1
-    },
-    "https://w3id.org/security/v2",
-    {
-      "JsonWebKey2020": {
-        "@id": "https://w3id.org/security#JsonWebKey2020"
+      "@version": 1.1,
+      "id": "@id",
```

Comment:
Should also have `"@protected": true` at the top level of this context.

Reply:
Fixed as well now.

Reply:
The usage of `@protected` ended up breaking many different tests in the BBS suite. I haven't been able to dig in and figure out what is the cause of this yet. The error I'm getting is that I'm redefining protected terms though.

Reply:
suggest opening an issue, and working to address it with better tests seperatly.

Reply:
@kdenhartog -- it sounds like adding `@protected` may have successfully surfaced errors that weren't getting detected previously in the BBS suite. The cause of those errors should be determined, we shouldn't turn off protection just to avoid them.